viewer/src/store.ts

```diff
@@ -125,7 +171,20 @@ const gaiaViewer = {
     // WRAPPER / DEBUG COMMUNICATION
     loadFromJSON(context, data: any) {},
   },
-  getters: {},
+  getters: {
+    currentRoundHexes: (state): GaiaHex[] => {
+      const data = state.data;
+      const roundEntryIndex = findLastIndex(data.advancedLog, (logItem) => Object.keys(logItem).includes("round"));
```

Comment:
```suggestion
      const roundEntryIndex = findLastIndex(data.advancedLog, (logItem) => "round" in logItem);
```

This is more efficient